test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -189,6 +189,18 @@ public void createTable() throws TableExistsException, AccumuloException,
     accumuloClient.tableOperations().delete(tableName);
   }
 
+  @Test
+  public void createTableWithTableNameLengthLimit()
+      throws AccumuloException, AccumuloSecurityException, TableExistsException {
+    StringBuilder tableNameBuilder = new StringBuilder();
+    for (int i = 0; i < 1026; i++) {
```

Comment:
@keith-turner I wasn't sure if we wanted a constant that was that long. I could just add it in as a 1025 char constant, since 1024 is the limit we are going for.

Reply:
@keith-turner Ignore my last comment. I was thinking about the actual test string.

Reply:
One nice thing about using a single constant in the code is that it makes it easier for someone to later search for all code that references that const.